src/argilla/client/api.py

```diff
@@ -112,10 +112,11 @@ def log(
     workspace: Optional[str] = None,
     tags: Optional[Dict[str, str]] = None,
     metadata: Optional[Dict[str, Any]] = None,
-    batch_size: int = 500,
+    batch_size: int = 100,
     verbose: bool = True,
     background: bool = False,
     chunk_size: Optional[int] = None,
+    num_threads: int = 0,
```

Comment:
Hi, thanks for fixing this. I would also like to see `max_retries` and the `backoff` variables here.

Reply:
Agree with `max_retries`, I forget to add it. For the backoff not sure what you want to expose really? I would like to start with an MVP allowing the setup of at least the retries before stopping. We can consider exposing other parameters later. wdyt @davidberenstein1957 ?

Reply:
`backoff` likely involves e.g. "expo", "constant", "fibo", etc. from https://github.com/litl/backoff
I think this is very overkill. Few people would ever change this from the default. I'd stick with `max_retries` as that one's easy to 1) use, 2) understand and 3) implement.

Reply:
IMO, we could add max_retries in combination with a default backoff 10, 20, 30